src/main/scala/com/gu/certificate/Magic.scala

```diff
@@ -90,15 +90,20 @@ object Magic extends BouncyCastle with FileHelpers {
     System.err.println(s"installing to IAM")
 
     val iamClient = region.createClient(classOf[AmazonIdentityManagementClient], installCredentialsProvider, null)
-    val uploadResult = iamClient.uploadServerCertificate(
-      new UploadServerCertificateRequest()
-        .withServerCertificateName(s"$safeDomain-exp$expDate")
-        .withPrivateKey(decryptedPem)
-        .withCertificateBody(certificatePem)
-        .withCertificateChain(chainPem)
-    )
-
-    System.err.println(s"successfully installed certificate in IAM as ${uploadResult.getServerCertificateMetadata.getArn}")
+    val certificateUploadRequest = new UploadServerCertificateRequest()
+      .withServerCertificateName(s"$safeDomain-exp$expDate")
+      .withPrivateKey(decryptedPem)
+      .withCertificateBody(certificatePem)
+      .withCertificateChain(chainPem)
+
+    Try {
+      val certificateUploadResult = iamClient.uploadServerCertificate(certificateUploadRequest)
+      val certificateArn = certificateUploadResult.getServerCertificateMetadata.getArn
+      System.err.println(s"successfully installed certificate in IAM as ${certificateArn}")
```

Comment:
For CLI apps I generally use the convention that if the output is potentially processable by another app then it should go to stdout. For text that would get in the way of parsing output then it should go to stderr. Take `ps` as an example - the column headers go to stderr and the data to stdout. Prism is similar, the host count and header go to stderr and the lines of hosts to stdout.

In this case there isn't really anything of use to send to stdout. Although the ARN of the cert is arguably one piece of information you might want to grab in a script.